aws/routes.go

```diff
@@ -37,6 +37,11 @@ func init() {
 		),
 		http.MethodPost: routes.H(postAwsAccount).With(
 			routes.RequestContentType{"application/json"},
+			routes.RequestBody{postAwsAccountRequestBody{
+				RoleArn:  "arn:aws:iam::123456789012:role/example",
+				External: "LlzrwHeiM-SGKRLPgaGbeucx_CJC@QBl,_vOEF@o",
```

Comment:
Is this external random ?

Reply:
It is an example.

Reply:
Okay, that was just to make sure (the `roleArn` looked like an example, but I wasn't sure about the external)